packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -57,6 +58,17 @@ export interface FieldConfig<TOptions extends object = any> {
   scopedVars?: ScopedVars;
 }
 
+export interface ValueLinkConfig {
+  /**
+   * Result of field reduction
+   */
+  calculatedValue?: FieldDisplay;
```

Comment:
Should we really allow an undefined calculatedValue?

Reply:
Well, here's the thing. In Gauge, Stat and BarGauge panel the visualisation can show a reduced vsalue, and only in such case this should be set. Otherwise its the row number of the field that's interesting. So yes, this should be allowed to be undef.

Reply:
Ok, thank you for that clarification

Reply:
Looking at how this is used inside the LinkSupplier.getLinks it only uses the display property, so maybe this should just be value: DisplayValue